GenderPayGap.WebUI/Views/ActionHubOld/Index.cshtml

```diff
@@ -41,46 +86,57 @@
     <div class="govuk-grid-column-one-third">
         <div class="guidance-related-content">
             <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible related-content-section-break">
-        
+
             <h3 class="govuk-heading-s">Related content</h3>
-            
+
             <ul class="govuk-list related-content-list">
                 <li>
-                    <a class="govuk-link govuk-body-s" href="@Url.Action("SearchResults", "Viewing")">
-                        Find and compare employer’s gender pay gaps
-                    </a>
-                </li>
-                <li>
-                    <a class="govuk-link govuk-body-s" href="/public/assets/pdf/understand-your-gender-pay-gap.pdf">
-                        Eight ways to understand your organisation's gender pay gap (PDF)
+                    <a class="govuk-link govuk-body-s" href="/public/assets/pdf/action-plan-guidance.pdf" rel="download">
+                        Four steps to developing a gender pay gap action plan
+                        <span class="govuk-visually-hidden"> (pdf)</span>
                     </a>
                 </li>
                 <li>
-                    <a class="govuk-link govuk-body-s" href="/public/assets/pdf/action-plan-guidance.pdf">
-                        Four steps to developing a gender pay gap action plan (PDF)
+                    <a class="govuk-link govuk-body-s" href="@Url.Action("EightWaysOverview", "EightWaysGuidance")">
+                        Eight ways to understand your oganisation's gender pay gap'
                     </a>
                 </li>
                 <li>
-                    <a class="govuk-link govuk-body-s" href="/public/assets/pdf/gender-pay-gap-explained.pdf">
-                        Gender pay gap explained (PDF)
+                    <a class="govuk-link govuk-body-s" href="@Url.Action("SearchResults", "Viewing")">
+                        Find and compare employer’s gender pay gaps

```

Comment:
Grammatically, I think this maybe should be `employers' gender pay gaps` since I read this as talking about multiple employers. May be reading it wrong, but probably best to check (probably with Claire) in case!